test/integration/controller/jobs/pod/pod_controller_test.go

```diff
@@ -319,6 +319,7 @@ var _ = ginkgo.Describe("Pod controller", ginkgo.Ordered, ginkgo.ContinueOnFailu
 							kueue.WorkloadEvictedByPreemption, "By test", "evict"),
 					).Should(gomega.Succeed())
 					util.FinishEvictionForWorkloads(ctx, k8sClient, createdWorkload)
+					util.ExpectEvictedWorkloadsTotalMetric(clusterQueue.Name, kueue.WorkloadEvictedByPreemption, 1)
```

Comment:
I think this [failure](https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_kueue/1955/pull-kueue-test-integration-main/1782940296334544896) is caused by this assert, because this integration test does not run scheduler, and the metric is bumped in scheduler code.

Reply:
You can cover this one [here](https://github.com/kubernetes-sigs/kueue/blob/36b340e396341c905444576bf797a221cb5f92ac/test/integration/scheduler/preemption_test.go#L139) instead with `util.ExpectEvictedWorkloadsTotalMetric(cq.Name, kueue.WorkloadEvictedByPreemption, 2)`